config/standard_ingest.yml.sample

```diff
@@ -4,6 +4,7 @@
         - Thumbs.db
         - metadata.txt
     :targets: dpc_targets
+    :intermediate_files: intermediate_files
```

Comment:
I'll let you know.  Removed the line from my local config file and re-running test suite.  Tests written for this functionality stub it but I suppose there might be other tests that will break.  If they do, I'll fix and update PR.